Library/Formula/libgpm.rb

```diff
@@ -0,0 +1,58 @@
+class Libgpm < Formula
+  desc "general purpose mouse"
+  homepage "http://www.nico.schottelius.org/software/gpm/"
+  url "http://www.nico.schottelius.org/software/gpm/archives/gpm-1.20.7.tar.bz2"
+  sha256 "f011b7dc7afb824e0a017b89b7300514e772853ece7fc4ee640310889411a48d"
+
+  head "https://github.com/telmich/gpm.git"
```

Comment:
Unless this formula also compiles correctly on Mac OS and you can test it, in which case send it upstream to Homebrew-core!